### PR TITLE
Fix console error when button is clicked & hover on button

### PR DESCRIPTION
**Pull request by RamuKammagani-osi**
#### Which User story does this relate to (link)?
- https://osi-cwds.atlassian.net/browse/COG-912

#### Brief feature description
-  Hover reflects a little and changes the color a bit

### CODE REVIEW

**Comment by Nasir2712** [comment]
I tried this locally, but the hover CSS background-color: none is an invalid value, can you check?

**Comment by RamuKammagani-osi** [comment]
> I tried this locally, but the hover CSS background-color: none is an invalid value, can you check?

**Comment by RamuKammagani-osi** [comment]
> I tried this locally, but the hover CSS background-color: none is an invalid value, can you check?

Updated

File: app/javascript/views/userDetail/UserDetailShow.jsx [hunk 1]
```diff
@@ -96,7 +96,7 @@ const UserDetailShow = ({
                             btnClassName="primary"
                             btnName="Resend Invite"
                             onClick={onResendInvite}
-                            disabled={disableResendEmailButton | disableEditBtn}
+                            disabled={disableEditBtn || disableResendEmailButton}
```
Comment (Nasir2712) [inline app/javascript/views/userDetail/UserDetailShow.jsx]: not sure why disable edit button prop is needed here?

Comment (RamuKammagani-osi) [reply app/javascript/views/userDetail/UserDetailShow.jsx]: Yes it works, But when the user is not editable resend button should be disabled, so i added  i added that

Comment (Nasir2712) [reply app/javascript/views/userDetail/UserDetailShow.jsx]: ok, I thought we hid the button, in that kind of scenario, anyways thanks for the clarification.

Comment (RamuKammagani-osi) [reply app/javascript/views/userDetail/UserDetailShow.jsx]: Yes, we are going to hide the button in the nearest future.

**Pull request merged**